### PR TITLE
Feature/tablet mvvm packaging

### PR DESCRIPTION
**Pull request by kejmil01**

### CODE REVIEW

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditViewModel.java [hunk 1]
```diff
@@ -4,35 +4,38 @@
...
+    private final TagEditionStartCallback tagEditionStartCallback;
     private final EditCompoundRecyclerView editCompoundRecyclerView;
     private TagsManager manager;
 
     EditViewModel(String title,
                   String imageUrl,
                   EditCompoundRecyclerView editCompoundRecyclerView,
                   TagsManager manager,
-                  EditTagCallback editTagCallback) {
+                  TagEditionStartCallback tagEditionStartCallback) {
```
Comment (BartSoftware) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditViewModel.java]: callback is enough

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/PhoneEditViewModel.java [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-    private PhoneEditTagCallback editTagCallback;
+    private PhoneTagEditionStartCallback editTagCallback;
 
     PhoneEditViewModel(
             String title,
             String imageUrl,
             EditCompoundRecyclerView editCompoundRecyclerView,
             TagsManager manager,
-            PhoneEditTagCallback editTagCallback) {
+            PhoneTagEditionStartCallback editTagCallback) {
```
Comment (BartSoftware) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/PhoneEditViewModel.java]: callback

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/ImagesViewModel.java [hunk 3]
```diff
@@ -18,7 +17,7 @@
     private final ImagesProvider imagesProvider;
     private final ImagesViewCallback imagesViewCallback;
 
-    ImagesViewModel(ImagesCompoundRecyclerView imagesCompoundRecyclerView,
+    public ImagesViewModel(ImagesCompoundRecyclerView imagesCompoundRecyclerView,
                     ImagesProvider imagesProvider,
                     ImagesViewCallback imagesViewCallback) {
```
Comment (BartSoftware) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/ImagesViewModel.java]: callback

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/phone/PhoneImagesViewModel.java [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
+import com.invisibleteam.goinvisible.mvvm.images.ImagesViewModel;
 import com.invisibleteam.goinvisible.mvvm.images.adapter.ImagesCompoundRecyclerView;
-import com.invisibleteam.goinvisible.mvvm.images.callback.PhoneImagesViewCallback;
 
 public class PhoneImagesViewModel extends ImagesViewModel {
 
     private PhoneImagesViewCallback imagesViewCallback;
 
-    PhoneImagesViewModel(ImagesCompoundRecyclerView imagesCompoundRecyclerView,
+    public PhoneImagesViewModel(ImagesCompoundRecyclerView imagesCompoundRecyclerView,
```
Comment (BartSoftware) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/phone/PhoneImagesViewModel.java]: recyclerView

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/phone/PhoneImagesViewModel.java [hunk 5]
```diff
@@ -1,15 +1,16 @@
...
                          ImagesProvider imagesProvider,
                          PhoneImagesViewCallback imagesViewCallback) {
```
Comment (BartSoftware) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/phone/PhoneImagesViewModel.java]: callback

**Review by BartSoftware — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditItemAdapter.java [hunk 6]
```diff
@@ -113,8 +113,8 @@ private boolean isTagChanged(Tag baseTag, Tag tag) {
         return false;
     }
 
-    void setOnTagActionListener(OnTagActionListener listener) {
-        this.onTagActionListener = listener;
+    void setEditViewModelCallback(EditViewModelCallback listener) {
```
Comment (kejmil01) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditItemAdapter.java]: Change parameter name to "callback"

**Review by BartSoftware — APPROVED** [review]

**Pull request merged**